configs/prophecy.toml

```diff
@@ -11,6 +11,7 @@ skip_samples = [
 ]
 skip_stages = ['SomalierPedigree']
 skip_samples_with_missing_input = true
+highmem_workers = true
```

Comment:
Is there a way to restrict that to only the failing stage (SampleQC), instead of applying this to all stages?
Should we maybe change this generally for the SampleQC stage in the code instead of the config?

Reply:
Good point! I think I can see what change needs to be made for this -- will address in a later PR :)